tests/basetest.js

```diff
@@ -14,12 +14,32 @@ const getUserToken = () => {
   const user = {
     username: 'prince',
     email: 'prince@gmail.com',
-    password: 'hello'
+    password: 'hello12345'
   };
   return chai
     .request(app)
     .post('/api/v1/auth/signup')
     .send(user);
 };
 
-export default getUserToken;
+const activateAccount = async () => {
+  await getUserToken()
+    .then(res => chai
+      .request(app)
+      .get(`/api/v1/activation/${res.body.user.id}`));
```

Comment:
When rebasing, remember to include the changes we made about getting a verified user without having to return the user id in the signup response body.

Reply:
solved